BUG: Context Menu - SQL Create is Not Working (Redshift)

**Describe the bug**
I'm connected to a Redshift database and when I right click on a table to bring up the context menu and click "SQL: Create" nothing happens. Bringing up the Developer Tools in BeekeeperStudio I see the following error.

```
vue.runtime.esm.js:1897 error: syntax error at or near "identity"
    at h.parseErrorMessage (app://./js/chunk-vendors.d7d1a4fa.js:54:386941)
    at handlePacket (app://./js/chunk-vendors.d7d1a4fa.js:54:383581)
    at h.parse (app://./js/chunk-vendors.d7d1a4fa.js:54:382227)
    at Socket.<anonymous> (app://./js/chunk-vendors.d7d1a4fa.js:139:35522)
    at Socket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:284:9)
    at Socket.Readable.push (internal/streams/readable.js:223:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)
```

**To Reproduce**
Steps to reproduce the behavior:
1. Connect to a Redshift Database
2. Right Click on a Table
3. Click "SQL: Create"
4. Nothing will happen
5. Open the Developer Tools and change to the console view
6. Try clicking "SQL: Create" again and you will see the error

**Expected behavior**
A new query window will open providing the SQL to re-create the table.

**Screenshots / Gifs**
Menu
<img width="173" alt="image" src="https://user-images.githubusercontent.com/7104326/181261674-f230634e-976d-4167-b2d8-bfdcfda002d7.png">

Error
<img width="539" alt="image" src="https://user-images.githubusercontent.com/7104326/181261027-2405869f-9711-4fd7-b024-4f9619167b30.png">

**Version Information (please complete the following information):**
 - OS: Mac v12.4
 - App Version 3.6.2
 - PostgreSQL 8.0.2 (Redshift 1.0.40182)

--- a/apps/studio/src/lib/db/clients/redshift.ts
+++ b/apps/studio/src/lib/db/clients/redshift.ts
@@ -6,7 +6,6 @@ import { FilterOptions, PrimaryKeyColumn, SupportedFeatures, TableOrView, TableP
 import { PostgresClient, STQOptions } from "./postgresql";
 import { escapeString } from "./utils";
 import pg from 'pg';
-import { defaultCreateScript } from "./postgresql/scripts";
 import { TableKey } from "@shared/lib/dialects/models";
 import { IDbConnectionServer } from "../backendTypes";
 import _ from "lodash";
@@ -179,14 +178,9 @@ export class RedshiftClient extends PostgresClient {
     }));
   }
   async getTableCreateScript(table: string, schema: string = this._defaultSchema): Promise<string> {
-    const params = [
-      table,
-      schema,
-    ];
+    const data = await this.driverExecuteSingle(`show table ${this.wrapIdentifier(schema)}.${this.wrapIdentifier(table)}`);
 
-    const data = await this.driverExecuteSingle(defaultCreateScript, { params });
-
-    return data.rows.map((row) => row.createtable)[0];
+    return data.rows.map((row) => row[data.columns[0].name])[0];
   }
 
   async createDatabase(databaseName: string, charset: string, _collation: string): Promise<void> {

